src/st/dai/background_block_init.c

```diff
@@ -8,7 +8,7 @@ ObjInit2 OVL_EXPORT(BackgroundBlockInit)[] = {{
     .zPriority = 0x1FA,
     .facingLeft = 0,
     .unk5A = 0,
-    .palette = PAL_DRA(0),
+    .palette = PAL_NONE,
```

Comment:
It makes sense to me to use the enum here also, but I'm not certain about it.  I can change it back if it isn't right.